src/Avalonia.Base/ValueStore.cs

```diff
@@ -33,6 +33,12 @@ public ValueStore(AvaloniaObject owner)
             _values = new AvaloniaPropertyValueStore<IValue>();
         }
 
+        public bool IsDuringInitialization
```

Comment:
Nit: Why `IsDuringInitialization` and not simply `IsInitializing` ?

Reply:
No particular reason, I can rename it.